src/main.rs

```diff
@@ -144,6 +146,52 @@ fn get_issue_tree_init_hashes(repo: &Repository, _: &clap::ArgMatches) -> i32 {
 
 // Porcelain subcommand implementations
 
+/// list subcommand implementation
+///
+fn list_impl(repo: &Repository, matches: &clap::ArgMatches) -> i32 {
+    // get initial commits
+    let mut commits : Vec<Commit>= try_or_1!(repo.get_all_issue_hashes())
+        .abort_on_err()
+        .map(|oid| repo.find_commit(oid))
+        .abort_on_err()
+        .collect();
+
+    // descending order, maybe limited to some number specified by the user
+    commits.sort_by(|a, b| b.time().cmp(&a.time()));
```

Comment:
I'm rather sure we can do this within the iteration above (before the `collect()` call) using the `itertools` crate.